aws-rds-dbparametergroup/src/main/java/software/amazon/rds/dbparametergroup/BaseHandlerStd.java

```diff
@@ -136,6 +136,26 @@ protected ProgressEvent<ResourceModel, CallbackContext> applyParameters(final Am
                 .then(progressEvent -> modifyParameters(progressEvent, currentDBParameters, proxy, proxyClient, requestLogger));
     }
 
+
+    protected ProgressEvent<ResourceModel, CallbackContext> applyParameters(final AmazonWebServicesClientProxy proxy,
+                                                                            final ProxyClient<RdsClient> proxyClient,
+                                                                            final ProgressEvent<ResourceModel, CallbackContext> progress,
+                                                                            final RequestLogger requestLogger) {
+        ResourceModel model = progress.getResourceModel();
+        CallbackContext callbackContext = progress.getCallbackContext();
```

Comment:
NIT: Let's keep marking these vars as immutable. This has became de-facto a standard in the project, I would prefer to keep it unified.

Reply:
Done. Made them `final`.